src/main/java/lotto/domain/Money.java

```diff
@@ -4,15 +4,23 @@
 
 public class Money {
 
-    private Integer money;
+    private final int money;
 
     public Money(int input) {
         validate(input);
         this.money = input;
     }
 
-    public Integer getMoney() {
-        return this.money;
+    public int getMoney() {
+        return new Integer(this.money);
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return this.money;
```